apps_download/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
```

Comment:
Everywhere Please use short headers such as https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py
(replacing year and Odoo Community Association)

Reply:
Done